dockerfiles/Dockerfile.openvino-csharp

```diff
@@ -47,7 +47,7 @@ ARG DEVICE=CPU_FP32
 ARG ONNXRUNTIME_REPO=https://github.com/microsoft/onnxruntime.git
 ARG ONNXRUNTIME_BRANCH=main
 
-ENV InferenceEngine_DIR=${INTEL_OPENVINO_DIR}/runtime/cmake
+ENV OpenVINO_DIR=${INTEL_OPENVINO_DIR}/runtime/cmake
```

Comment:
Can u remove this file. We no longer use and it is just going to add to confusion if we make change to  it

Reply:
Removed as it is no longer maintained here.